src/routes/search.js

```diff
@@ -9,6 +9,13 @@ router.post('/', loginOptional, paginate, async (ctx) => {
   ctx.body = await Search.find(ctx.request.body, { identityId: ctx.identity.id, shouldSave: !ctx.guest }, ctx.paginate)
 })
 
+router.post('/v2', loginOptional, paginate, async (ctx) => {
+  if (ctx.searchClient) {
```

Comment:
as it need the client to search this condition wont be good you should throw if it's not there

Reply:
Yeah, good point